src/types/Policy/Policy.ts

```diff
@@ -1,7 +1,7 @@
 import { Action } from './Actions';
 
 export interface Policy {
-    id?: number;
+    id: number;
```

Comment:
For policy creation we don't have the id in the UI yet. Wouldn't that break the creation?

Reply:
Mmh, it should break.
I'll check and revert this back.

Reply:
In the end I used a Policy with optional id for the creation pattern, this lets us assume the id is set everywhere else.